### ISSUE TITLE
Travel and stuff

### ISSUE THREAD

**Issue by pikesley** [opendatavillage/planning#1]
Just a thread to find out who's turning up when, really.

I plan to travel down on Thursday, probably with @chabrowa. There will be one more seat in the car if anybody wants it..

@Floppy @elsmorian @davea ?

**Comment by elsmorian** [comment]
Shotgun that seat @pikesley!

**Comment by pikesley** [comment]
OK, will you be able to get somewhere East-ish on the Thursday?

**Comment by davea** [comment]
Ooh, I didn't realise being a village gave us early access. I'll come down on Thursday evening too in that case.

**Comment by pikesley** [comment]
We're not really registered as a full-on village, but then I'm not sure we were last time either, and we got in a day early

**Comment by davea** [comment]
shotgun your sofa when we get told to come back on Friday then 😉

**Comment by pikesley** [comment]
Yeah, I think I'm going to amend this, and instead head out _very early_ Friday morning

**Comment by elsmorian** [comment]
Oh aye? why the change?
